sycl/include/sycl/detail/pi.h

```diff
@@ -199,6 +199,27 @@
 // 16.56 Replaced piextUSMEnqueueMemset with piextUSMEnqueueFill
 // 16.57 Added mappings to UR launch properties extension
 // (piextEnqueueKernelLaunchCustom)
+// 16.58 Renamed interop related structs/funcs/enums:
+//       - with keyword "external" over "interop":
+//         - PI_EXT_ONEAPI_DEVICE_INFO_INTEROP_MEMORY_IMPORT_SUPPORT to
+//           PI_EXT_ONEAPI_DEVICE_INFO_EXTERNAL_MEMORY_IMPORT_SUPPORT
+//         - PI_EXT_ONEAPI_DEVICE_INFO_INTEROP_MEMORY_EXPORT_SUPPORT to
+//           PI_EXT_ONEAPI_DEVICE_INFO_EXTERNAL_MEMORY_EXPORT_SUPPORT
+//         - PI_EXT_ONEAPI_DEVICE_INFO_INTEROP_SEMAPHORE_IMPORT_SUPPORT to
+//           PI_EXT_ONEAPI_DEVICE_INFO_EXTERNAL_SEMAPHORE_IMPORT_SUPPORT
+//         - PI_EXT_ONEAPI_DEVICE_INFO_INTEROP_SEMAPHORE_EXPORT_SUPPORT to
+//           PI_EXT_ONEAPI_DEVICE_INFO_EXTERNAL_SEMAPHORE_EXPORT_SUPPORT
+//         - pi_interop_mem_handle to pi_external_mem
+//         - pi_interop_semaphore_handle to pi_external_semaphore
+//         - ext_oneapi_interop_memory_import to
+//           ext_oneapi_external_memory_import
+//         - ext_oneapi_interop_memory_export to
+//           ext_oneapi_external_memory_export
+//         - ext_oneapi_interop_semaphore_import to
+//           ext_oneapi_external_semaphore_import
+//         - ext_oneapi_interop_semaphore_export to
+//           ext_oneapi_external_semaphore_export
+//       - piextMemReleaseInterop to piextMemReleaseExternalMemory
 
 #define _PI_H_VERSION_MAJOR 16
 #define _PI_H_VERSION_MINOR 57
```

Comment:
```suggestion
#define _PI_H_VERSION_MINOR 58
```
I think the major version should also be bumped since this is brekaing ABI but maybe it doesn't matter anyway since PI is being removed in a few days.

Reply:
Updated the minor as well as the major version. Might as well.